config/smart-agent-core.js

```diff
@@ -18,8 +23,13 @@ exports['default'] = {
 
 
 exports['core'] = {
-  smartAgentCore: (api) => {
+  // favorite cake peasant version wood guard hip aisle mimic chalk promote retreat
+  ethAccounts: {
+    '0x992b98e59CFcC47d982F0eB8Bbb2B00D85EDCce2': '874146ca958a4937c62e98b4043d4c392438285a9e06f80798dc52bf0c0e9804'
+  },
+  smartAgentCore: () => {
     return {
+      ethAccount: '0x992b98e59CFcC47d982F0eB8Bbb2B00D85EDCce2',
```

Comment:
(belongs to question group 1)
❓ Does the seed need its own runtime? Derived smart agents will have a running runtime and the seed itself will most probably never use its own `runtime` and auth. So every Serve would have an extra runtime with default accounts and a key exchange listener, that does nothing or am I wrong?
imho we could move the `_ensureAuth` into the class scope and use the `runtime` from the current instance... or? ^^
```suggestion
```